packages/@orchidui-vue/src/Form/Button/OCButton.vue

```diff
@@ -28,88 +26,144 @@ const Icon = defineAsyncComponent(() =>
 );
 const props = defineProps({
   label: String,
-  isSecondary: Boolean,
-  isDestructive: Boolean,
-  isSmall: Boolean,
-  isLarge: Boolean,
   isDisabled: Boolean,
   isLoading: Boolean,
-  icon: String,
-  iconAfter: String,
-  isRounded: Boolean,
-  isText: Boolean,
+  leftIcon: String,
+  rightIcon: String,
+  isTransparent: Boolean,
+  isShortcut: Boolean,
+  variant: {
+    type: String,
+    default: "primary",
+  },
+  size: {
+    type: String,
+    default: "default",
+  },
 });
+const isIconOnly = computed(
+  () => (props.leftIcon || props.rightIcon) && !props.label,
+);
 
-const buttonClassName = computed(() => {
-  let className = "border-1 border-oc-primary-500 ";
-
-  if (props.isDestructive) {
-    className += props.isText
-      ? "text-oc-error "
-      : "oc-btn-error text-oc-text-000 ";
-  } else if (props.isSecondary) {
-    className += props.isText
-      ? "text-oc-text-400 "
-      : "oc-btn-secondary text-oc-text-400 ";
-  } else {
-    className += props.isText
-      ? "text-oc-primary "
-      : "oc-btn-primary  text-oc-text-000 ";
-  }
+const buttonTypeClasses = computed(() => ({
+  primary: !props.isTransparent
+    ? "border border-oc-blue oc-btn-primary text-white"
```

Comment:
please move oc blue to primary color, we need button color customize, also add rounded full varianf (now not ready in figma i will ask to UI/UX , but in store design we need it )